src/core.js

```diff
@@ -56,11 +58,14 @@ extend( QUnit, {
 			var module = {
 				name: moduleName,
 				parentModule: parentModule,
-				tests: []
+				tests: [],
+				testsRun: 0
```

Comment:
Looking at the ["events details types" issue](https://github.com/js-reporters/js-reporters/issues/30), I think we would be able to replace `module.testsRun` with `module.status.total` when it is implemented since it looks like it will be similar.

I would also be fine with setting an equivalent property on the `tests` array, though it feels a little less correct.